crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -252,34 +252,43 @@ def cancel
   api_version "2.0"
   example '
   {
-    "ceph": {
-      "available": false,
-      "repos": {}
-    },
     "ha": {
-      "available": false,
-      "repos": {}
-    },
-    "os": {
       "available": true,
-      "repos": {}
+      "repos": [
+        "SLE12-SP2-HA-Pool",
+        "SLE12-SP2-HA-Updates"
+      ],
+      "errors": {
+      }
     },
-    "openstack": {
+    "os": {
       "available": false,
-      "repos": {
+      "repos": [
+        "SLES12-SP2-Pool",
+        "SLES12-SP2-Updates"
+      ],
+      "errors": {
         "missing": {
           "x86_64": [
-            "SUSE-OpenStack-Cloud-8-Pool",
-            "SUSE-OpenStack-Cloud-8-Updates"
+            "SLES12-SP2-Pool"
           ]
         },
         "inactive": {
           "x86_64": [
-            "SUSE-OpenStack-Cloud-8-Pool",
-            "SUSE-OpenStack-Cloud-8-Updates"
+            "SLES12-SP2-Pool"
           ]
         }
       }
+    },
+    "openstack": {
+      "available": true,
+      "repos": [
+        "Cloud",
+        "SUSE-OpenStack-Cloud-7-Pool",
```

Comment:
ah, ok, github's "context detection" fails for docstrings above `def functionname` :)